app/services/importers/ok_sos/base_importer.rb

```diff
@@ -9,7 +9,7 @@ class BaseImporter < ApplicationService
 
       def initialize(file_path)
         @file_path = file_path
-        @model_cache = ActiveSupport::HashWithIndifferentAccess.new({})
+        @@id_cache = ActiveSupport::HashWithIndifferentAccess.new({})
```

Comment:
Have to be extra careful with the cache now since it persists across imports.
Did this since the initial load for Entities etc takes a while.